examples/plot/main.go

```diff
@@ -32,9 +32,8 @@ func loop() {
 		g.Plot("Plot 基本图表").AxisLimits(0, 100, -1.2, 1.2, g.ConditionOnce).XTicks(lineTicks, false).Plots(
 			g.Line("Plot Line 线图", linedata),
 			g.Line("Plot Line2", linedata2),
-			g.SwitchPlotAxes(g.AxisX1, g.AxisY2),
 			g.Scatter("Scatter 散点图", scatterdata),
```

Comment:
Turns out that the problem is on this line.
SwitchPlotAxes is not possible unless you set up these axes.
I'll try to figure out how to do that.